.circleci/config.yml

```diff
@@ -1804,97 +1805,87 @@ jobs:
             - test-unit-harness-cpu-pycov
 
   test-unit-harness-cpu-tf1:
-    machine:
-      image: <<pipeline.parameters.machine-image>>
-    resource_class: gpu.nvidia.small
+    docker:
+      - image: determinedai/environments:py-3.7-pytorch-1.7-tf-1.15-cpu-6eceaca
+    resource_class: determined-ai/container-runner-gpu
     steps:
       - checkout
       - skip-if-docs-only
       - skip-if-github-only
       - skip-if-webui-only
-      - setup-docker-env:
-          image: determinedai/environments:py-3.7-pytorch-1.7-tf-1.15-cpu-6eceaca
-          image-name: test_container
-          extras-requires: "mypy pytest coverage"
-          gpus: false
-      - run: mkdir -p $HOME/.local/bin
       - install-codecov
       - run: echo 'export PATH=$PATH:$HOME/.local/bin' >> $BASH_ENV
-      - run: docker exec -e COVERAGE_FILE=/mnt/project/test-unit-harness-cpu-tf1-pycov test_container make -C harness test-cpu-tf1
-      - run: docker exec test_container coverage xml -i --data-file=./test-unit-harness-cpu-tf1-pycov
+      - run: echo 'export PATH=$PATH:/usr/local/nvidia/bin' >> $BASH_ENV
+      - run: echo 'export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/nvidia/lib64' >> $BASH_ENV
```

Comment:
Is this related to LD_LIBRARY_PATH being broken in these images? I would just bump it to the latest environment image versions rather than setting it here, I think [this bug was fixed](https://github.com/determined-ai/environments/pull/210).

Reply:
This is to ensure the nvidia libraries are loaded when calling any nvidia binary.

Reply:
Is this not the case in our current environments? Because that sounds like a bug to me.

Reply:
It's dependent on the running instance, not what's baked into the image.
This is moreso about getting access to the nvidia libraries mounted by k8s in the backend instead of "fixing" the image.